nservicebus/upgrades/5to6.md

```diff
@@ -234,6 +234,7 @@ The `IMessageMutator` and `IMutateTransportMessages` interfaces are no longer av
 
  * `IncomingContext` and `OutgoingContext` have been replaced with multiple contexts ("stages") per pipeline offering more useful information depending on the requirements. [Pipeline Steps Stages and Connectors](/nservicebus/pipeline/steps-stages-connectors.md) provides more details about the available stages.
  * `Behavior<TContext>` now provides the base class for implementing custom behaviors and replaces `IBehavior<TContext>`.
+ * Behavior instances are created once, when the endpoint starts up, and then cached for the life of the endpoint. This is true even if the behavior type is registered as `InstancePerCall` or `InstancePerUnitOfWork`.
```

Comment:
OK, I like your proposition of change, so I see two options:

1. for now we'll go with 

"This is true even if the behavior type is registered using `RegisterComponents` method as `InstancePerCall` or `InstancePerUnitOfWork`."

and add note to the issue in NSB repo

2. we'll mark this as WIP and clarify after the NSB issue is resolved

I'd say it depends on when do we expect the code change will get done.